templates/documentation/dynamic_indicators.html

```diff
@@ -0,0 +1,481 @@
+{% extends 'base.html' %} {% block content %}
+<h2>Dynamically Generated Indicators from Downscaled CMIP6</h2>
+
+<p>
+    This endpoint provides access to dynamically generated indicators derived from
+    a 6-model average of CMIP6 daily data that have been bias-adjusted and downscaled
+    using dynamically downscaled ERA5 reference data. The source dataset spans 
+    1965&ndash;2100 at 4km spatial resolution. The spatial domain covers most of Alaska 
+    and parts of western Canada. Variables include daily maximum and minimum temperature
+     (<code>tasmax</code> and <code>tasmin</code>), as well as daily total precipitation 
+     (<code>pr</code>).
+</p>
+
+<h3>Available Operations</h3>
+<table class="variable-table">
+  <thead>
+    <tr>
+      <th>Operation</th>
+      <th><code>route</code></th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Count annual days above / below threshold</td>
+      <td>
+        <code>count_days/above/&lt;value&gt;/&lt;units&gt;/...</code>, 
+        <code>count_days/below/&lt;value&gt;/&lt;units&gt;/...</code>
+      </td>
+      <td>
+        Counts number of days per year where values are above or below the provided <code>&lt;value&gt;</code>
+        given in the provided <code>&lt;units&gt;</code>. For area queries, daily values are weighted by spatial 
```

Comment:
I think the second sentence here could use more explanation: what do you mean weighted by spatial area?